src/test/java/jewellery/inventory/helper/ResourceTestHelper.java

```diff
@@ -69,6 +69,21 @@ public static Resource getPearl() {
         .build();
   }
 
+  public static Resource getPearl(BigDecimal price) {
+    return Pearl.builder()
+            .id(RESOURCE_ID)
+            .clazz(CLAZZ_PEARL)
+            .quantityType(QUANTITY_TYPE_UNIT)
+            .type(TYPE_PEARL)
+            .size(SIZE)
+            .quality(QUALITY)
+            .color(COLOR_PEARL)
+            .shape(SHAPE_PEARL)
+            .pricePerQuantity(price)
+            .note(NOTE)
+            .build();
+  }
```

Comment:
Now there is code repetition. Can you edit the other method to be:
```
public static Resource getPearl() {
    return getPearl(PRICE_PER_QUANTITY);
}
```

Reply:
done!